lib/src/Uncertainty/Algorithm/Classification/MixtureClassifier.cxx

```diff
@@ -77,6 +78,48 @@ UnsignedInteger MixtureClassifier::classify(const NumericalPoint& inP) const
   return bestClass;
 }
 
+Indices MixtureClassifier::classify(const NumericalSample & inS) const
+{
+  if (inS.getDimension() != getDimension()) throw InvalidArgumentException(HERE) << "Error: the sample to classify has dimension=" << inS.getDimension() << " but the classifier expects dimension=" << mixture_.getDimension();
+  if (ResourceMap::GetAsBool("Classifier-Parallel"))
+    return classifyParallel(inS);
+  return classifySample(inS);
+}
+
+Indices MixtureClassifier::classifySample(const NumericalSample & inS) const
```

Comment:
There is no need to redefine classify, just rename classifySample into classifySequential.